setup.py

```diff
@@ -87,9 +87,9 @@
 
 
 install_requires = [
-    'BeautifulSoup>=3.2.1,<3.3.0',
```

Comment:
need to identify if this was required, and if it is, we either need to use BS 4 or lxml.html (or some other dep).